content/en/docs/spin-operator/tutorials/scaling-with-hpa.md

```diff
@@ -11,44 +11,71 @@ Horizontal scaling, in the Kubernetes sense, means deploying more pods to meet d
 
 ## Prerequisites
 
-> We use k3d to run a Kubernetes cluster locally as part of this tutorial, but you can follow these steps to configure HPA autoscaling on your desired Kubernetes environment.
-
 Please see the following sections in the [Prerequisites]({{< ref "prerequisites" >}}) page and fulfil those prerequisite requirements before continuing:
 
-- [Go]({{< ref "prerequisites#go" >}})
 - [Docker]({{< ref "prerequisites#docker" >}}) - for running k3d
 - [kubectl]({{< ref "prerequisites#kubectl" >}}) - the Kubernetes CLI
 - [k3d]({{< ref "prerequisites#k3d" >}}) - a lightweight Kubernetes distribution that runs on Docker
+- [Helm]({{< ref "prerequisites#helm" >}}) - the package manager for Kubernetes
 - [Bombardier]({{< ref "prerequisites#bombardier" >}}) - cross-platform HTTP benchmarking CLI
 
-## Fetch Spin Operator (Source Code)
+> We use k3d to run a Kubernetes cluster locally as part of this tutorial, but you can follow these steps to configure HPA autoscaling on your desired Kubernetes environment.
+
+## Setting Up Kubernetes Cluster
 
-If you haven't already, please go ahead and clone the Spin Operator repository and move into the Spin Operator directory:
+Run the following command to create a Kubernetes cluster that has [the containerd-wasm-shims](https://github.com/deislabs/containerd-wasm-shims) pre-requisites installed: If you have a Kubernetes cluster already, please feel free to use it:
```

Comment:
Should this link to the spinkube version of the shim now?

Reply:
Good question. Does the k3d image come with `containerd-shim-spin` only? then I would also vote for changing the link (and the caption) to `[containerd-shim-spin](https://github.com/spinkube/containerd-shim-spin)`.

However, if the k3d image contains multiple shims, I would suggest keeping it as it is.

Reply:
The k3d image (eg `ghcr.io/spinkube/containerd-shim-spin/k3d:v0.12.0`) from spinkube/containerd-shim-spin does only come with the `containerd-shim-spin-v2` binary included/configured.